go/cmd/vtctldclient/command/vreplication/materialize/create.go

```diff
@@ -77,6 +82,24 @@ should be copied as-is from the source keyspace. Here's an example value for tab
 			if err := common.ParseAndValidateCreateOptions(cmd); err != nil {
 				return err
 			}
+
+			var hasTableSettings, isReference bool
+			if createOptions.TableSettings.val != nil {
+				hasTableSettings = true
+			}
+			if common.CreateOptions.IsReference && len(common.CreateOptions.Tables) > 0 {
+				isReference = true
+			}
+			switch {
+			case !hasTableSettings && !isReference:
+				return vterrors.Errorf(vtrpcpb.Code_FAILED_PRECONDITION, missingCreateParams)
+			case hasTableSettings && isReference:
+				return vterrors.Errorf(vtrpcpb.Code_FAILED_PRECONDITION, "cannot specify both --table-settings and --reference/--tables")
+			case common.CreateOptions.IsReference && len(common.CreateOptions.Tables) == 0:
+				return vterrors.Errorf(vtrpcpb.Code_FAILED_PRECONDITION, "cannot specify --reference without --tables")
+			case !common.CreateOptions.IsReference && len(common.CreateOptions.Tables) > 0:
+				return vterrors.Errorf(vtrpcpb.Code_FAILED_PRECONDITION, "cannot specify --tables without --reference")
```

Comment:
If both options are required, why not replace them with a single `--reference-tables` option? It will still be mutually exclusive with `--table-settings`.

Reply:
I had kept them as two options as 
* `--tables` felt more consistent as we have a simlar `MoveTables` option
* In case we would have other use cases similar to `--reference` where we would materialize a list of special tables.

But I don't really have a clear scenario in mind. Will switch to the single option.

Reply:
Modified to use single option. Added to release notes.